test/JDBC/input/schema_resolution_func-vu-prepare.sql

```diff
@@ -0,0 +1,41 @@
+create schema schema_resolution_func_s1;
+go
+
+create table schema_resolution_func_t1(a int);
+go
+
+create table schema_resolution_func_s1.schema_resolution_func_t1(a int);
+go
+
+insert into schema_resolution_func_s1.schema_resolution_func_t1 values (1);
+go
+
+create function dbo.schema_resolution_func_f1()
```

Comment:
we should have tests which have procedures/functions/views and triggers nested at 3-4 levels to validate correct search path is used. Also, we should test TVF, MSTVF etc. as well

Reply:
Added more tests in the next commit.